src/invoices/InvoiceDetails/BatchGroupValue/BatchGroupValue.js

```diff
@@ -0,0 +1,54 @@
+import React, {
+  useState,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  KeyValue,
+  Spinner,
+} from '@folio/stripes/components';
+
+import { batchGroupByPropResource } from '../../../common/resources';
+
+const BatchGroupValue = ({ id, label, mutator }) => {
+  const [batchGroup, setBatchGroup] = useState();
+
+  useEffect(
+    () => {
+      setBatchGroup();
+
+      if (id) {
+        mutator.invoiceBatchGroup.GET()
+          .then(setBatchGroup)
+          .catch(() => setBatchGroup({}));
+      }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [id],
+  );
+
+  if (!batchGroup) {
+    return <Spinner />;
```

Comment:
why it's spinner if we use `Loading` in other places?

Reply:
we use `<Icon icon="spinner-ellipsis" />` but [Spinner](https://github.com/folio-org/stripes-components/blob/1769842a305e80b96d635a458d021c37b29d76fa/lib/Spinner/Spinner.js#L4) it's the same